EIPS/eip-7779.md

```diff
@@ -0,0 +1,91 @@
+---
+eip: 7779
+title: Add Controlled Gas Target Increase Strategy
+description: Adds a controlled gas target increase strategy.
+author: Giulio Rebuffo (@Giulio2002)
+discussions-to: https://ethresear.ch/t/eip-7779-discussion-thread/20574
```

Comment:
Please create a discussions topic in Eth Magicians.  https://ethereum-magicians.org/c/eips/5

Reply:
yes - i finally managed to